tests/test_compare_lightgbm.py

```diff
@@ -0,0 +1,50 @@
+try:
+    import lightgbm as lb
+except ImportError:
+    exit(0)
```

Comment:
Instead of doing this, please use:

```python
lb = pytest.importorskip("lightgbm")
```

at the beginning of each of the tests that use lb.